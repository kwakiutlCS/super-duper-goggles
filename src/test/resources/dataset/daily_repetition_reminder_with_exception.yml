reminder:
- id: 1
  content: content
  user_id: user
  created_at: 1000
  updated_at: 1000
- id: 2
  content: content
  user_id: user
  created_at: 1000
  updated_at: 1000
time:
- reminder_id: 1
  timestamp: 3600 
  step: 1
  unit: DAYS
  bound_type: NO_BOUND
  zone: Z
  exceptions: 90000
- reminder_id: 2
  timestamp: 3600 
  step: 1
  unit: DAYS
  bound_type: NO_BOUND
  zone: Z
  exceptions: 90000