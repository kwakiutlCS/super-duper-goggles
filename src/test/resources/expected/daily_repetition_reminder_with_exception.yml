reminder:
- id: 1
  content: content
  user_id: user
  created_at: 1000
  updated_at: 1000
time:
- reminder_id: 1
  timestamp: 3600 
  step: 1
  unit: DAYS
  bound_type: NO_BOUND
  zone: Z
  exceptions: 3600